docs/search/solr_search_engine.md

```diff
@@ -33,6 +33,10 @@ cp -R ./vendor/ibexa/solr/src/lib/Resources/config/solr/* server/ez/template
 cp server/solr/configsets/_default/conf/{solrconfig.xml,stopwords.txt,synonyms.txt} server/ez/template
 cp server/solr/solr.xml server/ez
 
+# If you are using Ibexa Commerce, additionally copy commerce-specific configuration files:
+cat ../vendor/ibexa/commerce-shop/src/bundle/Search/Resources/config/solr/custom-fields-types.xml >> server/ez/template/custom-fields-types.xml
+cat ../vendor/ibexa/commerce-shop/src/bundle/Search/Resources/config/solr/language-fieldtypes.xml >> server/ez/template/language-fieldtypes.xml
```

Comment:
```suggestion
cat [IbexaDXP_installdir]/vendor/ibexa/commerce-shop/src/bundle/Search/Resources/config/solr/language-fieldtypes.xml >> server/ez/template/language-fieldtypes.xml
```

Reply:
If we change this as suggested, don't forget to also fix line 30 : 
```diff
-cp -R ./vendor/ibexa/solr/src/lib/Resources/config/solr/* server/ez/template
+cp -R [IbexaDXP_installdir]/vendor/ibexa/solr/src/lib/Resources/config/solr/* server/ez/template
```
( I am not able to make a suggestion directly in this review as that line hasn't been changed